### PR TITLE
Create Blog “announcing-chapel-1-30-0”

### PR DESCRIPTION
**Pull request by bradcray**
Automatically generated by Netlify CMS

### CODE REVIEW

**Comment by bradcray** [comment]
This was my first time attempting to set up a card to an external Chapel blog, and apparently I failed.  I've sent mail to @reddypramod85 requesting assistance and insight.

File: content/blog/announcing-chapel-1-30-0.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+title: Announcing Chapel 1.30.0!
+date: 2023-03-24T00:29:50.615Z
+priority: ""
```
Comment (choukden) [inline content/blog/announcing-chapel-1-30-0.md:4]: Hi Brad, 

I think I understand the reason for the CMS rendering process failure: the Priority option was probably set to a value then removed.

Using the CMS editor, select your blog link entry, and the could you please set the priority to a value higher than 1. Say 2 or higher value.

Save, then click "Writing in Blog collection / Changes saved" to exit the CMS editor mode and return to the Editorial Workflow. Then let the rendering process to run for a few minutes (4 to 5 minutes).

 This workaround should make the trick.

 I will then merge the PR in github to push the blog post link to the HPE Dev community portal.

Best regards,

Denis

CC: @reddypramod85

**Review by choukden — CHANGES_REQUESTED** [review]
Hi Brad,

Thank you very much for your new blog link entry.
I provide you with a tip to work around the issue you have been experiencing with the creation of the blog link.

Best regards
Denis

File: content/blog/announcing-chapel-1-30-0.md (hunk 1 above)
Comment (bradcray) [reply content/blog/announcing-chapel-1-30-0.md]: Thanks Denis — I think you're right that I selected a priority and then attempted to unselect it from the UI.  I've followed your instructions and we'll see how the CI does this time.

Comment (bradcray) [reply content/blog/announcing-chapel-1-30-0.md]: @choukden : Looks clean this time around—let me know if there's more I need to do here.

**Pull request merged**